cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -153,10 +153,10 @@ func GetEtcdPodSpec(cfg *kubeadmapi.InitConfiguration, initialCluster []etcdutil
 func getEtcdCommand(cfg *kubeadmapi.InitConfiguration, initialCluster []etcdutil.Member) []string {
```

Comment:
what about changing the `getEtcdCommand` signature similarly to other `get*commands`?

Reply:
NB. if I'm not wrong, this unblock changing also the signature for `GetEtcdPodSpec` and `CreateStackedEtcdStaticPodManifestFile`